packages/gluestick/src/types.js

```diff
@@ -30,35 +61,34 @@ export type LoggerTypes = {
   error: Function;
 }
 
-export type ProjectConfig = {
-
-};
-
-type WebpackConfigEntry = string | string[] | Object;
-
-export type WebpackConfig = {
-  [key: string]: WebpackConfigEntry,
+export type Context = {
+ config: Config;
+ logger: Logger;
 };
 
-export type GSConfig = {
+export type UniversalWebpackConfigurator = (options: any) => WebpackConfig;
 
-};
+export type Question = {
+  type: string;
+  name: string;
+  message: string;
+}
 
 export type CreateTemplate = (
+  interpolations: Array<*>,
   strings: Array<string>,
-  interpolations: Array<*>
 ) => (args: Object) => string;
 
 export type Generator = {
-  name: string,
-  config: any,
+  name: string;
+  config: any;
 }
 
 export type WrittenTemplate = {
-  name: string,
-  config: any,
+  name: string;
+  config: any;
```

Comment:
This should be done in separate issue, besides I didn't wrote those types, I only replaced commas with semicolons.